linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -83,7 +83,18 @@ private void run(String bomCoordinates)
     ImmutableSet<SymbolProblem> problemsInSnapshot =
         LinkageChecker.create(snapshot).findSymbolProblems().keySet();
 
+    Set<SymbolProblem> fixedErrors = Sets.difference(problemsInBaseline, problemsInSnapshot);
     Set<SymbolProblem> newErrors = Sets.difference(problemsInSnapshot, problemsInBaseline);
+
+    if (newErrors.equals(fixedErrors)) {
+      System.out.println(
+          "Snapshot versions have the same " + problemsInBaseline.size() + " errors as baseline");
+      return;
+    }
+    if (!fixedErrors.isEmpty()) {
+      System.out.println("The following errors are fixed:");
```

Comment:
How about this? It's slightly clearer: "The following errors in the baseline no longer appear in the snapshot:"